tools/proteomiqon_mzmltomzlite/proteomiqon_mzmltomzlite.xml

```diff
@@ -0,0 +1,292 @@
+                        <conditional name="PaddingParamsCond">
+                            <param type="boolean" name="usePaddingparams" label="Padding" checked="false"/>           
+                            <when value="true">
+                                <conditional name="MaximumPaddingPointsCond">
+                                    <param type="boolean" name="specifyMaximumPaddingPoints" label="Speficy maximum padding points" checked="true"/>
+                                    <when value="true">
+                                        <param type="integer" name="MaximumPaddingPoints" label="Maximum padding points" value="0"/>
+                                    </when>        
+                                </conditional>
+                                <param type="float" name="Padding_MzTolerance" label="Padding MZ tolerance" value="0"/>
+                                <param type="integer" name="WindowSize" label="Window size" value="0"/>
+                                <param type="float" name="SpacingPerc" label="Spacing perc" value="0"/>
+                            </when>
+                        </conditional>
+                    </when>
+                </conditional>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data format="sqlite" name="out_mzlite" />
+    </outputs>
+    <help>
+    <![CDATA[
+Introduction
+------------
+The success of modern proteomics was made possible by constant progression in the field of mass spectrometry. 
+Over the course of the past years quite a few manufacturers of mass spectrometers have managed to establish themselfes in the field of biological research. 
+Since aquisition and accession of mass spectra are performance critical processes, various performance optimized, but vendor specific and closed source formats have been developed to store raw MS data. 
+This comes to the disadvantage for toolchain developers which want to provide tools for every scientist regardless of the format of their raw data.
+
+In a effort to provide an open format for the storage of MS data the format mzML was developed. While this XML based format is straight forward to implement it falls behind in performance critical scenarios. 
+To be competitive in performance and to comply to the FAIR principles we chose to use mzLite, an open and SQLite based implementation of mzML, within our toolchain.
+
+What It Does
+------------
+The tool mzMLToMzLite allows to convert mzML files to mzLite files. Additionally, it allows the user to perform peak picking or filtering of mass spectra.
+We recommend the use of `_msconvert <https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/msconvert/msconvert/3.0.20287.2>`_ to convert your raw data into .mzML. 
+
+Further Reading
+---------------
+Additional information about the tool can be found in the `_documentation <https://csbiology.github.io/ProteomIQon/tools/MzMLToMzLite.html>`_.
+    ]]>
+    </help>
```

Comment:
you can add here a citation if you like

please run `planemo lint` against this tool to see what should be added